binding/python/_orca.py

```diff
@@ -279,36 +440,75 @@ def synthesize(
         Custom pronunciations can be embedded in the text via the syntax `{word|pronunciation}`.
         The pronunciation is expressed in ARPAbet format, e.g.: "I {live|L IH V} in {Sevilla|S EH V IY Y AH}".
         :param speech_rate: Rate of speech of the synthesized audio.
```

Comment:
minor but it might be helpful to state the accepted range here e.g. 0.7-1.3 inclusive, and that a higher number is faster